intTests/support/find-java-rt-jar.sh

```diff
@@ -0,0 +1,17 @@
+#! /bin/bash
+
+# Find the Java core classes JAR, 'rt.jar' (called 'classes.jar' on OS
```

Comment:
This is a bit unrelated to the spirit of the PR itself, but it feels a bit unnecessary to manually search for `rt.jar` now that `saw` supports finding `rt.jar`'s location by way of `--java-bin-dirs`/`PATH`. Perhaps we should open an issue as a reminder to see if we can avoid having to use this script.

Reply:
I've opened #1061 for this.